### ISSUE TITLE
[BOUNTY] Algorand - Best In-Theme Application 🎨

### ISSUE THREAD

**Issue by wisthsb1519** [ethberlinzwei/Bounties#15]
# Algorand: Best In-Theme Application 🎨

Algorand is the best way to start building your decentralized application quickly! Algorand lets you build any type of layer 2 application on a blockchain network with near-instant transaction finality. Our JavaScript, Python, Go and Java [SDK’s](https://developer.algorand.org/docs/using-sdks-and-rest-apis) allow you to take advantage of identity, key, account and wallet management easily and effectively for your webapp.

## How to qualify

Qualifying submission should 
-    make use of the [developer tools](https://developer.algorand.org/docs/getting-started)
-    Code must be posted on Github
-    comply with the overall ETHBerlin submission guideline

## Resources

Example solutions:
-   [Evancoin](https://medium.com/algorand/l2-applications-on-algorand-make-your-own-coin-6d62c5d5578d)
-   [ARC-01](https://medium.com/assetblock/arc-01-building-a-token-standard-proposal-on-algorand-17fe634397ce)

Community Resources:
-   [Algorand Developer Documentation](https://developer.algorand.org/)
-   [Algorand Forums](https://forum.algorand.org/)
-   [Gitter Community](https://gitter.im/algorand-ethzweihack/community)
-   [Discord](https://discord.gg/SqNRnw7)
-   [Reddit](htps://www.reddit.com/r/algorand/)

## Judging Criteria

In order to qualify for prizes, teams must demonstrate creative competence by creating a solution using our dev tools that promotes the principles of the Bauhaus Art movement. The Bauhaus manifesto stipulates that art should be building, making no distinction between artist and artisan- so in that spirit, break the border of art and science - become both artist and blockchain developer. 

1\.     Creative: How creative or innovative is the application?
2\.     Technicals: What technical stack does the application make use of? How well is it built, technically?
3\.     Application: How applicable is the solution to the real world?
4\.     Complete: How complete is this application with respect to the Algorand dev tools?

Examples with the theme:

- a solution that cryptographically verifies art
- use a camera-enabled raspberry pi to capture data from a physical piece of artwork and trigger txns on-chain if the artwork is from The Met Art Collection  using The Met’s API https://metmuseum.github.io/
- basically find someway to capture art or art-related data, write that data to the notefield using one our SDKs, create/sign/broadcast txn
- Use your imagination 🌈

## Prizes

1\.     🥇First Place: Gotenna x 2
2\.     🥈Second Place: Ledger Nano
